src/components/Type2020/Blueprint.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
+ *
+ *
+ * Name proposals
+ * --------------------------
+ * Type refers to the current soon to be "legacy" Type components.
+ * We need something to differentiate from our current Type component while transitioning.
+ *   - Type2020 (working prototype in this doc & files)
+ *   - Type2020 / TwentyType
+ *   - TypeNew / NewType
+ *   - TrendyType
+ *   - ???
+ *
+ * Should we append the new name to the end of Type2020 components like below?
+ * TitleSmall vs TitleSmall2020
+ *
+ * Should we rename Type to TypeLegacy or similar?
+ *
+ *
+ * Problems to solve
+ * --------------------------
+ * Provide access for developers to the new Type styles while maintaining backwards compatability.
+ *
+ * Allow for a slow rollout to minimize the risk of things breaking.
+ *
+ * Make transitioning as seamless as possible, existing components should work without any developer
+ * interference aside from setting a new import name for the component.
+ * Ex: import Type2020.TitleXLarge as TitleXLarge
```

Comment:
`import { TitleXLarge2 } from Type2` or, we go with `import { Type2 } from Type2` and then do via props e.g. `<Type2 type={title-x-large} ...` or something

Reply:
I'm open to looking into this route forward.